pyproject.toml

```
@@ -18,20 +18,32 @@ rpm_repo_name_prefix = "inmanta/oss"
 image_registry = "ghcr.io"
 image_repository = "inmanta"
 image_name = "oss"
+image_tags = [
+    "dev",
+    "{major_version}-dev",
+]
```

I am not sure whether it will useful to have the `{major_version}-dev` tag in practice, because the major version on OSS doesn't mean a lot. It just the year in which the version was released. @bartv Wdyt?
```suggestion
image_tags = [
    "dev",
]
```

---

true. oss is more of a stream of releases where there is no such thing as major or minor.